Tests/Core/Cache.cs

```diff
@@ -56,8 +56,8 @@ public void StoreRetrieve()
             FileAssert.AreEqual(file, cached_file);
         }
 
-        [Test, TestCase("cheesy.zip","cheesy.zip"), TestCase("Foo-1:2.3","Foo-1-2.3"),
-            TestCase("Foo-1:2:3","Foo-1-2-3"), TestCase("Foo/../etc/passwd","Foo-..-etc-passwd")]
+        [Test, TestCase("cheesy.zip","cheesy.zip"), TestCase("Foo-1-2.3","Foo-1-2.3"),
+            TestCase("Foo-1-2-3","Foo-1-2-3"), TestCase("Foo-..-etc-passwd","Foo-..-etc-passwd")]
```

Comment:
Just to confirm, we've changed these tests because the assumption is that we'll always pass hints into `cache.Store` which don't result in invalid filenames?

That seems reasonable, although I might pop an assert into the code just to make it clear this is the case. :)